integration_tests/tests/jinja_helpers/test_slugify.sql

```diff
@@ -1,6 +1,6 @@
 {% if dbt_utils.slugify('!Hell0 world-hi') == 'hell0_world_hi' %}
     {# Return 0 rows for the test to pass #}
-    select 1 limit 0
+    select 1 {{ limit_zero }}
```

Comment:
the error doesn't lend itself to this being the problem, but I noticed that the other places that use {{ limit_zero }} do alias the column so let's give that a go. 

```suggestion
    select 1 as col_name {{ limit_zero }}
```